### PR TITLE
Completed es6-classes.

### PR DESCRIPTION
**Pull request by Youssef-Najjarine**
image
image
image

### CODE REVIEW

File: es6-classes/example-class.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class ExampleClass {
+  constructor(example) {
+    this.example = example;
+  }
```
Comment (scbowler) [inline es6-classes/example-class.js:4]: The `ExampleClass` should be empty, a `constructor` is not required.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, one small update needed. Let me know if you have any questions.

File: es6-classes/example-class.js (hunk 1 above)
Comment (Youssef-Najjarine) [reply es6-classes/example-class.js]: image

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**